controllers/cloud.redhat.com/providers/inmemorydb/redis_test.go

```diff
@@ -37,7 +37,7 @@ func TestLocalRedis(t *testing.T) {
 		RedisDeployment: &dd,
 		RedisService:    &svc,
 	}
-	_ = makeLocalRedis(&env, objMap, true, false)
+	_ = makeLocalRedis(&env, &env, objMap, true, false)
```

Comment:
We pass in env twice, but the second param is supposed to be a ClowdObject. I'm sure I just don't understand but was this on purpose?

Reply:
Yes this was on purpose - the first time is the "I'll always give you an env", the second is - and "the redis you create will be for this env" - as opposed to ..... for this particular "app"

Reply:
A ClowdObject is either a ClowdApp or a ClowdEnvironment - interface FTW

Reply:
Gotcha thanks!